### PR TITLE
Update English translations. Added Simplified Chinese Translations. Updated translations for all other languages.

### PR DESCRIPTION
**Pull request by zhangguiyu**
Dear Alisu,

Thank you for creating a great inspectional based registration module. We are using it in our product for the Chinese market. I have added simplified Chinese, and also made minor changes to a few English descriptions, e.g., "Registration has accepted" ---changed to--> "Registration accepted", and also updated the admin UI to include "Activation Key Expired" translations.

Please review the changes and accept/reject as you deem appropriate.

Yours Truly,
Kuiyu

### CODE REVIEW

File: src/registration/locale/de/LC_MESSAGES/django.po [hunk 1]
```diff
@@ -7,413 +7,412 @@ msgid ""
...
 "MIME-Version: 1.0\n"
 "Content-Type: text/plain; charset=UTF-8\n"
 "Content-Transfer-Encoding: 8bit\n"
 "Plural-Forms: nplurals=2; plural=(n != 1);\n"
-"Last-Translator: Patrick Husi <patrick.husi@arpage.ch>\n"
-"Language-Team: \n"
 "X-Generator: Poedit 1.8.2\n"
 
-#: src/registration/forms.py:66
+#: admin/__init__.py:262
```
Comment (lambdalisue) [inline src/registration/locale/de/LC_MESSAGES/django.po]: Well I am not really sure but why `src/registration/` prefix is omitted? Do you have idea? Maybe the version difference of the django (then it's ok)?

Comment (zhangguiyu) [inline src/registration/locale/de/LC_MESSAGES/django.po]: That prefix is unnecessary as django i18n will use the base of the app as the root path, in this case, the base path is registration/

If you look at other apps, e.g., django-taggit

/pythonenv/lib/python2.7/site-packages/taggit/locale/ja/LC_MESSAGES

#: models.py:138
msgid "Tagged Item"
msgstr "タグ付け済みのアイテム"

#: models.py:139
msgid "Tagged Items"
msgstr "タグ付け済みのアイテム一覧"

They also typically omit the prefix.

Comment (lambdalisue) [inline src/registration/locale/de/LC_MESSAGES/django.po]: Make sense. Thanks ;-)

**Comment by lambdalisue** [comment]
LGTM

**Comment by lambdalisue** [comment]
Thanks a lot ;-)

**Pull request merged**